### DISCUSSION TITLE
Is there a way to increase the amount of time logins are valid for?

### DISCUSSION THREAD

**Discussion by DarrienG** [stringer-rss/stringer#606] [Q&A] [answered]
Stringer is a great app, but I've found the validity of logins is rather short especially for mobile. It looks like it uses a browser session cookie: `Expires / Max-Age:"Session"`

For browsing on a mobile phone or instance, the built in task manager will just kill the browser every so often and require a re-login which can be a little annoying.

Is there any way to set the cookie to last for like a week or a month instead?

**Comment by mockdeep** [comment]
@DarrienG yeah, I've been meaning to look into that. It's a little annoying on desktop, too. I'm not sure when I'll get a chance to work on it, but if you're interested in having a stab at it I'll be happy to look at a PR.

**Comment by mockdeep** [comment, marked as answer]
Okay, I just merged [a PR extending sessions to 2 weeks](https://github.com/stringer-rss/stringer/pull/889). If you pull the latest `main` you should be able to stay logged in for longer.
